include/range/v3/range_fwd.hpp

```diff
@@ -393,12 +393,11 @@ namespace ranges
         struct ordered_less;
         struct ident;
 
-        enum cardinality
+        enum cardinality : std::ptrdiff_t
```

Comment:
The two features - enum classes and explicit underlying types - both came into the language at the same time, so people often conflate them.

> AFAIK This is not an enum base ...
The actual name of the grammar non-terminal in the Standard that matches an explicit underlying type declaration is [*enum-base*](http://eel.is/c++draft/dcl.enum#nt:enum-base).